app/views/placeholder/landing.html.erb

```diff
@@ -34,8 +34,11 @@
           <%=image_tag("LEAD logo.png", :align=> "middle", :style => "width:300px")%>
           <h1>Communities@Cal</h1>
           <p>Please login through CalNet to continue.</p> 
-          <%= form_tag login_path, :method => 'post' do %>
-            <input id="login-button" type="submit" value="Placeholder Login" class="btn-primary"/>
+          <%= form_tag signin_path, :method => 'get' do %>
+          <div class="col-md-4 offset-md-4">
+            <input id="login-button" type="submit" value="Sign in" class="btn btn-primary btn-block"
+                   style="background: linear-gradient(#2683bc, #1f7099), #1f7099"/>

```

Comment:
Ideally we should not do inline style to help with maintainability, but not a blocker for this feature!